aten/src/ATen/cpu/vec256/vec256_bool.h

```diff
@@ -0,0 +1,331 @@
+#pragma once
```

Comment:
can you pull out the vec256_bool stuff into another PR?

uint8 is strange because sometimes we treat it as a an actual uint8 and sometimes we treat it as a bool, and that's something we should fix in the future.  From a quick read of this I can't tell which parts implement the uint8_t behavior and which parts implement the bool behavior.  Is there bool-specific behavior here?

Reply:
Sure, let me pull it out.